src/postgres/commands/configurePostgresFirewall.ts

```diff
@@ -38,7 +39,8 @@ export async function setFirewallRule(context: IActionContext, treeItem: Postgre
     const resourceGroup: string = nonNullProp(treeItem, 'resourceGroup');
     const serverName: string = nonNullProp(treeItem, 'azureName');
 
-    const firewallRuleName: string = "azureDatabasesForVSCode-publicIp";
+    // firewall rule name on the portal is restricted to 80 characters or less
+    const firewallRuleName: string = "azDbVSCode-Ip" + `-${randomUtils.getPseudononymousStringHash(ip, 'hex')}`;
```

Comment:
How long is the result of `getPseudononymousStringHash`? We normally just add 6-10 random characters to make names unique, otherwise the names are pretty unreadable. That's what I've seen the portal doing, too.

Reply:
It is way longer than 10, more like 60 or something. I agree it should be smaller, what I am worried about is duplicity. Right now, the publicIP is hashed to hex. Maybe once hashed I can just use part of the hash for the label. I am guessing this would still make it unique.

Reply:
I would probably just do `getRandomHexString(6)`. If you're worried about that creating duplicates, I think there's only a 1 in 16 million chance of that happening (16^6 = 16,777,216). Plus the names only have to be unique within a certain subscription, right? So virtually impossible for dupes.

I'm not sure how hashing the IP affects the probabilities, so that's the only reason I would lean towards avoiding `getPseudononymousStringHash`, but honestly I think it's also virtually impossible for duplicates.

Reply:
That's true, I haven't thought about that. Using just a small string should work.